src/helpers/database/schema.sql

```diff
@@ -122,6 +122,17 @@ CREATE TABLE follows (
   INDEX created (created)
 );
 
+CREATE TABLE walletfollows (
```

Comment:
We shouldnt make a new table for it, we should instead use "follows" table, but add a field "type", type can be "space" or "account", later we can also merge "subscription" there.

Reply:
I'd call it "following"

Reply:
And add a column "type"